mods/ra/rules/ai.yaml

```diff
@@ -82,7 +82,7 @@ Player:
 				CheckRadius: 10c0
 			Consideration@2:
 				Against: Enemy
-				Types: Vehicle, Tank, Infantry, Defense
+				Types: Vehicle, Tank, Infantry, Defense, Water
```

Comment:
I saw a lot of boats (some are high value cruisers) at https://github.com/OpenRA/OpenRA/issues/6636#issue-44622273 so I suggest this will let the AI avoid them.